public/js/index.js

```diff
@@ -72,8 +87,9 @@ $(function () {
     objNewWidget.position.my = `${$('#widgetPositionMyX').val()} ${$('#widgetPositionMyY').val()}`
     objNewWidget.position.at = `${$('#parentPositionAtX').val()} ${$('#parentPositionAtY').val()}`
     objNewWidget.id = getNextId()
-    console.log(objNewWidget)
+    //console.log(objNewWidget)
```

Comment:
Either this line should be completely removed, or we need to write a set of utility functions for various levels of logging. The ROS world uses DEBUG, INFO, WARNING, and ERROR, so it would be nice to duplicate that.

Reply:
done